packages/node-cli/src/commands/account/setupci/AccountSetupCiValidation.js

```diff
@@ -0,0 +1,123 @@
+const {
+	showValidationResults,
+	validateFieldIsNotEmpty,
+	validateFieldHasNoSpaces,
+	validateAlphanumericHyphenUnderscore,
+	validateMaximumLength,
+} = require('../../../validation/InteractiveAnswersValidator');
+const ProjectInfoService = require('../../../services/ProjectInfoService');
+const assert = require('assert');
+const { throwValidationException } = require('../../../utils/ExceptionUtils');
+const NodeTranslationService = require('../../../services/NodeTranslationService');
+
+const COMMAND = {
+	OPTIONS: {
+		ACCOUNT: 'account',
+		AUTHID: 'authid',
+		CERTIFCATEID: 'certificateid',
+		PRIVATEKEYPATH: 'privatekeypath',
+		DOMAIN: 'domain',
+
+	},
+	SDK_COMMAND: 'authenticateci',
+};
+//url is a not mandatory parameter for setup mode
+const URL_PARAM = 'url';
+//Select indicator, the action behaves in two different ways depending on select parameter is selected or not
+const SELECT_PARAM = 'select';
+//not mandatory params in setup (not select) mode, all the other parameters are mandatory
+const NOT_MANDATORY_STANDARD_MODE_PARAMS = [COMMAND.OPTIONS.DOMAIN, URL_PARAM];
+const { COMMAND_OPTIONS : { IS_MANDATORY, IS_MANDATORY_SELECT_MODE, IS_NOT_ALLOWED_SELECT_MODE} } = require('../../../services/TranslationKeys');
+
+class AccountSetupCiValidation {
+	constructor(commandMetadata, runInInteractiveMode) {
+		this._commandMetadata = commandMetadata;
+		this._runInInteractiveMode = runInInteractiveMode;
+	}
+
+	validateAuthID(authId) {
```

Comment:
Can we rename method to `validateAuthIDFormat` / `isValidAuthIDFormat` or similar?

Reply:
My main point was to add `AuthIDFormat` to the validation name to make clear what is the validation about.
I gave a couple of suggestions one with validate verb, can you change the method name to `validateAuthIDFormat` or something that states what is the validation about?